.github/ISSUE_TEMPLATE/tech-issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Tech Issue
+about: Engineering-specific technical work that is not product-specific. Engineering team "owns" these issues.
+title: ''
+labels: tech
```

Comment:
```suggestion
labels: `tech`
```
nittiest nit 😄

Reply:
updating, as there's a proper syntax for this